Mathlib/Data/List/EditDistance/Estimator.lean

```diff
@@ -23,10 +23,8 @@ This is then used in the implementation of `rewrite_search`
 to avoid needing the entire edit distance calculation in unlikely search paths.
 -/
 
-set_option autoImplicit true
-
 variable {α β δ : Type} [CanonicallyLinearOrderedAddCommMonoid δ]
```

Comment:
I know you didn't edit this, but this missing `*` is intentional?

Reply:
Good question! I checked; \alpha can be generalised, but making \beta or \delta take Type* errors all over the place. I didn't think much if that can be fixed, but will leave that to a follow-up.